content/docs/administration/permissions.md

```diff
@@ -24,41 +24,43 @@ The following table lists the area and name of the permissions in the central se
+| Permissions (3)      | Requisitions          | View requisitions                   | Can view Requisitions                                                                                        |
+|                      |                       | Create and edit requisitions        | Can maintain Requisitions                                                                                    |
+| omSupply Permissions |                       | Confirm Internal Order sent         | Can confirm Internal Orders as sent                                                                          |
+| Permissions (3)      | Stocktakes            | Create Stocktake                    | Can create Stocktakes                                                                                        |
+|                      |                       | Delete Stocktake                    | Can delete Stocktakes                                                                                        |
+|                      |                       | Add Stocktake lines                 | Can add Stocktake lines                                                                                      |
+|                      |                       | Edit Stocktake lines                | Can edit Stocktake lines                                                                                     |
+| Permissions (3)      | Vaccines              | View sensor details                 | Can view sensor details                                                                                      |
+|                      |                       | Edit sensor location                | Can edit sensor location                                                                                     |
+| omSupply Permissions |                       | Cold chain API access               | Can access the Cold Chain API                                                                                |
+| Permissions (3)      | Assets                | View assets                         | Can view Assets                                                                                              |
+|                      |                       | Add, edit assets                    | Can maintain Assets                                                                                          |
+|                      |                       | Setup Assets                        | Can import Asset catalogue items                                                                             |
+| omSupply Permissions | Immunization Programs | Can modify central data             | Can modify the Immunization Programs and their Vaccine Courses                                               |
+| omSupply Permissions | Demographics          | Can modify central data             | Can modify Demographic Indicators                                                                            |
```

Comment:
Hi @jmbrunskill  - you had the minimum permissions in your PR for all of GAPS, which ones do you think are important to capture here?